scss/_header.scss

```diff
@@ -660,7 +660,7 @@
     transform 0.4s ease, 
     filter 0.4s ease, 
     box-shadow 0.4s ease;
-    animation: gradient-pulse 3s infinite;
+    animation: none;
     @keyframes gradient-pulse {
```

Comment:
Please delete the keyframe related to the animation and check if its not used in another place 
thanks